admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -1046,6 +1023,14 @@ $product-page-padding-bottom: 80px !default;
   }
 }
 
+.product-page-v2 {
```

Comment:
should we use `v2` inside css that will stay?

Reply:
The idea is to have a "scope" that only targets the v2 page, because for now the css is used in both pages which can result in conflicts and break the original page

Once the legacy page is removed I guess we could remove this extra class and merge it with the regular one but for now it's quite handy, and it allows to see clearly what we have changed (sadly we didn't do this previously so we already merged quite a lot of stuff in the initial css)